packages/engine/src/scene/components/MediaComponent.ts

```diff
@@ -192,6 +192,15 @@ export function MediaReactor() {
 
   if (!isClient) return null
 
+  function validateTime() {
+    const mediaElementState = getMutableComponent(entity, MediaElementComponent)
```

Comment:
i'd rename mediaElementState to mediaElementComponent here

Reply:
implemented